amxmodx/amxmodx.cpp

```diff
@@ -1541,7 +1542,8 @@ static cell AMX_NATIVE_CALL register_concmd(AMX *amx, cell *params) /* 4 param *
 	CmdMngr::Command* cmd;
 	int access = params[3];
 	bool listable = true;
-	
+	bool info_ml = *params / sizeof(cell) >= 6 && params[6] != 0 && i;
```

Comment:
There are no booleans, the params array is always of type **cell** which is in turn an **int**.